cpp/tests/copying/slice_tests.cpp

```diff
@@ -344,3 +344,20 @@ TEST_F(SliceTableCornerCases, NegativeOffset)
 
   EXPECT_THROW(cudf::slice(src_table, indices), cudf::logic_error);
 }
+
+TEST_F(SliceTableCornerCases, MiscOffset)
```

Comment:
It’s already merged now but I think it’d have made sense as a copy range test rather than slice test

Reply:
I will add/change the test case in one of my follow-up PRs